docker-for-windows/edge-release-notes.md

```diff
@@ -24,7 +24,7 @@ notes](release-notes). For Docker Desktop system requirements, see
 
 ### Known issue
 
-Windows Insider Preview Slow Ring users cannot run WSL 2 after upgrading to Docker Desktop Edge 2.1.6.1 release as WSL 2 requires Windows 10 Insider Preview build 19018 or higher.
+Windows Insider Preview Slow Ring users with OS older than build 19025 cannot run WSL 2 after upgrading to Docker Desktop Edge 2.1.6.1 release. WSL 2 requires Windows 10 Insider Preview build 19018 or higher.
```

Comment:
Perhaps reword to read: 

Windows Insider Preview Slow Ring users running OS builds older than 19025 cannot run WSL 2 following upgrade to Docker Desktop Edge 2.1.6.1 (WSL 2 requires Windows 10 Insider Preview build 19018 or greater).

Reply:
Thanks Kory. I've updated the note.